CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+# CMakeLists files in this project can
+# refer to the root source directory of the project as ${HRPD_SOURCE_DIR} and
+# to the root binary directory of the project as ${HRPD_BINARY_DIR}
+cmake_minimum_required(VERSION 3.22.1)
+project (HRPD)
+
+add_compile_definitions(EIGEN_DONT_PARALLELIZE)
+
+include_directories(include)
+find_package(OpenMP)
+
+list(PREPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
+
+# Libigl
+include(libigl)
+
+# Enable the target igl::glfw
+igl_include(glfw)
+
+# Set compilation flags
+set(OpenMP_CXX_FLAGS "-fopenmp")
```

Comment:
With MSVC I got `unknown option -fopenmp`. Changing this to `/openmp` worked there. Could you put this into a `if (MSVC) ... ` condition?

Reply:
Done! I didn't know that that was an option thank you for the recommendation!